### ISSUE TITLE
Undefined index: size at core/lib/private/Files/View.php#1287

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#29705]
### Steps to reproduce
1. Run integration tests, e.g.
```
./run.sh federation_features/federated.feature:225
```
2. Notice the "Undefined index" messages like:
```
[Wed Nov 29 03:52:52 2017] Undefined index: size at /home/phil/core/lib/private/Files/View.php#1287
[Wed Nov 29 03:52:52 2017] 127.0.0.1:41130 [201]: /remote.php/webdav/PARENT%20(2)/testquota.txt-olddav-regular
[Wed Nov 29 03:52:53 2017] 127.0.0.1:41154 [201]: /remote.php/webdav/PARENT%20(2)/testquota.txt-olddav-oldchunking-chunking-42-2-0
[Wed Nov 29 03:52:55 2017] 127.0.0.1:41174 [201]: /remote.php/webdav/PARENT%20(2)/testquota.txt-olddav-oldchunking-chunking-42-2-1
[Wed Nov 29 03:52:56 2017] Undefined index: size at /home/phil/core/lib/private/Files/View.php#1287
[Wed Nov 29 03:52:56 2017] 127.0.0.1:41202 [201]: /remote.php/dav/files/user1/PARENT%20(2)/testquota.txt-newdav-regular
[Wed Nov 29 03:52:56 2017] 127.0.0.1:41230 [201]: /remote.php/dav/uploads/user1/chunking-43
[Wed Nov 29 03:52:56 2017] 127.0.0.1:41232 [201]: /remote.php/dav/uploads/user1/chunking-43/1
[Wed Nov 29 03:52:57 2017] 127.0.0.1:41234 [201]: /remote.php/dav/uploads/user1/chunking-43/2
[Wed Nov 29 03:52:58 2017] Undefined index: size at /home/phil/core/lib/private/Files/View.php#1287
```
or look in a Jenkins log for similar.

### Expected behaviour
No "Undefined index" messages.

### Actual behaviour
"Undefined index" messages.

### Server configuration
Current 10.0.4 ccode from git running in local dev environment or in CI.

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @PVince81.

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `status/STALE` is set and there were no updates for 7 days. Feel free to reopen this issue if you deem it appropriate.

(This is an automated comment from [GitMate.io](https://gitmate.io).)

**Comment by phil-davis** [comment]
Fixed by #29706 and backported to stable10 for 10.0.5 in PR #29743
